astropy/cosmology/_io/mapping.py

```diff
@@ -295,20 +321,20 @@ class (e.g. "FlatLambdaCDM") or the class itself.
 
 
 def to_mapping(
-    cosmology,
-    *args,
-    cls=dict,
-    cosmology_as_str=False,
-    move_from_meta=False,
-    rename=None,
-):
+    cosmology: Cosmology,
+    *args: Any,
```

Comment:
If this function must accept `*args` but doesn't use them at all then `object` would be a better annotation than `Any`.

Reply:
✅